### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Eclipse Azure Sign In (Dark Theme)_Color Contrast: In Dark theme, Color contrast ratio for the ‘Error message’ and the ‘Dialog name’ in blue color is less than 4.5:1

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3315]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.3

**Note: Similar issue is found for all the popups where the text under title is blue color and for the error messages**

**Repro Steps:** 
1) Open Java EE - Eclipse application and change the theme to dark in (WindowsPreferencesGeneralappearancesSelect Dark theme)
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Verify ‘Azure Sign In’ text in blue color with black background has required color contrast
5) Select Service principle radio button and Tab till Sign In button and press enter
6) Verify the color contrast ratio for the error message

**Actual Result:** 
In Dark theme, Color contrast ratio for the ‘Error message’ and the ‘Dialog name’ in blue color is 4.2:1 which is less than 4.5:1

**Expected Result:** 
In Dark theme, Color contrast ratio for the ‘Error message’ and the ‘Dialog name’ in blue color should be greater than or equal to 4.5:1

**User Impact:** 
Users with low vision will not be able to see the text properly if the color contrast ratio is less than required

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_3_color contrast fails for error message](https://user-images.githubusercontent.com/41109135/62203460-da4f0480-b3a8-11e9-89f7-bd4106150580.png)
![Bug_3_color contrast fails for the dialog name](https://user-images.githubusercontent.com/41109135/62203461-da4f0480-b3a8-11e9-9104-9e30bea32c2c.png)
![Screenshot (346)](https://user-images.githubusercontent.com/41109135/62211067-39b61000-b3bb-11e9-8240-d736d4b7aadf.png)
![Screenshot (344)](https://user-images.githubusercontent.com/41109135/62211068-39b61000-b3bb-11e9-8497-5bfffedff827.png)
![Screenshot (343)](https://user-images.githubusercontent.com/41109135/62211069-3a4ea680-b3bb-11e9-9ddb-bd2e5ffdb437.png)

**Comment by Flanker32** [comment]
All of the above component are inherited from Eclipse `TitleAreaDialog`, which is out of the scope of the Azure Toolkit, and the color is affected by Eclipse theme, and eclipse provide multiple theme and this issue will not happen in other theme, for example, Light, here are the screen shots.

*Sign in:*
![image](https://user-images.githubusercontent.com/12445236/63935678-91b26600-ca90-11e9-907c-b21b6f41755d.png)

*Create dynamic web project:*
![image](https://user-images.githubusercontent.com/12445236/63935866-f968b100-ca90-11e9-9195-44c5b45b5e65.png)

*Select Subscription:*
![image](https://user-images.githubusercontent.com/12445236/63935877-ff5e9200-ca90-11e9-8bf6-d6ef15b10862.png)

In all above screenshots, the front color of error message is black and the background color id white, with ratio 21:1.
![image](https://user-images.githubusercontent.com/12445236/63935941-2a48e600-ca91-11e9-8a9e-96f313d537de.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium,  Users with low vision will not be able to see the error message properly.
- Quality of experience: not impacted. Eclipse provide alter theme for users, for instance, light, which has more clear fonts for error messages.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati  Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Tracking in AAT [here](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please have a review.

**Comment by Tanushagottipati** [comment]
External Bug:https://bugs.eclipse.org/bugs/show_bug.cgi?id=550841

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749740
